Chef/Cookbooks/Relativity/recipes/post_relativity_update_relativity_logos.rb

```diff
@@ -0,0 +1,29 @@
+custom_log 'custom_log' do msg 'Starting Updating Relativity Logos' end
+    start_time = DateTime.now
+    custom_log 'custom_log' do msg "recipe_start_time(#{recipe_name}): #{start_time}" end
+    
+    admin_image_path = win_friendly_path(File.join(Chef::Config[:file_cache_path], '\cookbooks\Relativity\files\default\Relativity-Button-Admin.png'))
+    small_image_path = win_friendly_path(File.join(Chef::Config[:file_cache_path], '\cookbooks\Relativity\files\default\Relativity-Button.png'))
+    relativity_login_image_path = win_friendly_path(File.join(Chef::Config[:file_cache_path], '\cookbooks\Relativity\files\default\relativityLogin.png'))
+    destination_admin_image_path = "C:\\Program Files\\kCura Corporation\\Relativity\\EDDS\\Images\\Relativity-Button-Admin.png"
+    destination_samll_image_path = "C:\\Program Files\\kCura Corporation\\Relativity\\EDDS\\Images\\Relativity-Button.png"
+    destination_relativity_loging_image_path = "C:\\Program Files\\kCura Corporation\\Relativity\\EDDS\\Images\\relativityLogin.png"
+
+    # Install Disclaimer Acceptance Log Rap
+    custom_log 'custom_log' do msg 'Updating Relativity Logos' end
+    
+    powershell_script 'update_relativity_logos' do
+      code <<-EOH
+        Copy-Item  -Path #{admin_image_path} -Destination "#{destination_admin_image_path}" -Force
```

Comment:
check if the image exists. if exists, do replace. If it not exists, throw an exception.

you can use test-path to check if the file exists. 
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/test-path?view=powershell-6